templates/web/src/payload.ts.twig

```diff
@@ -0,0 +1,40 @@
+export class Payload extends Blob {
+    public name: string | null = null;
+
+    constructor(data: BlobPart[], name?: string) {
+        super(data);
+        this.name = name || null;
+    }
+
+    public async toString() {
+        return await this.text();
+    }
+
+    public async toJson<T = unknown>(): Promise<T> {
+        return JSON.parse(await this.text());
+    }
+
+    public async toBinary(): Promise<Uint8Array> {
+        return new Uint8Array(await this.arrayBuffer());
+    }
+
+    public async toFile(): Promise<File> {
+        return new File([this], this.name || "file");
```

Comment:
We should not have default file names. Let it be null/udefined

Reply:
File objects must have a name